example/conf.py

```diff
@@ -0,0 +1,142 @@
+### Project information
+############################################################
+
+# Product name
+project = "Example Project"
+author = "Example Project Authors"
+
+# The title you want to display for the documentation in the sidebar.
+# You might want to include a version number here.
+# To not display any title, set this option to an empty string.
+html_title = project + " Docs"
+
+# The default value uses the current year as the copyright year.
+#
+# For static works, it is common to provide the year of first publication.
+# Another option is to give the first year and the current year
+# for documentation that is often changed, e.g. 2022–2023 (note the en-dash).
+#
+# A way to check a GitHub repo's creation date is to obtain a classic GitHub
+# token with 'repo' permissions here: https://github.com/settings/tokens
+# Next, use 'curl' and 'jq' to extract the date from the GitHub API's output:
+#
+# curl -H 'Authorization: token <TOKEN>' \
+#   -H 'Accept: application/vnd.github.v3.raw' \
+#   https://api.github.com/repos/canonical/<REPO> | jq '.created_at'
+copyright = f"{datetime.today().year}, {author}"
+
+# Set the path to your own static assets here
+html_static_path = ["_static"]
+
+# (Some settings must be part of the html_context dictionary, while others
+#  are on root level. Don't move the settings.)
+html_context = {
+    # Change to the link to the website of your product (without "https://")
+    # For example: "ubuntu.com/lxd" or "microcloud.is"
+    # If there is no product website, edit the header template to remove the
+    # link (see the readme for instructions).
+    "product_page": "github.com/example/project",
+    # Add your product tag (the orange part of your logo, will be used in the
+    # header) to ".sphinx/_static" and change the path here (start with "_static")
+    # (default is the circle of friends)
+    "product_tag": "_static/example-tag.png",
+    # Change to the discourse instance you want to be able to link to
+    # using the :discourse: metadata at the top of a file
+    # (use an empty value if you don't want to link)
+    "discourse": "https://discourse.example-project.com",
+    # Change to the GitHub URL for your project
+    "github_url": "https://github.com/example/project",
+    # Change to the branch for this version of the documentation
+    "github_version": "v9.9.9",
+    # Change to the folder that contains the documentation
+    # (usually "/" or "/docs/")
+    "github_folder": "/example/docs/",
+    # Change to an empty value if your GitHub repo doesn't have issues enabled.
+    # This will disable the feedback button and the issue link in the footer.
+    "github_issues": "enabled",
+    # Controls the existence of Previous / Next buttons at the bottom of pages
+    # Valid options: none, prev, next, both
+    "sequential_nav": "none",
+}
+
+# If your project is on documentation.ubuntu.com, specify the project
+# slug (for example, "lxd") here.
+slug = "example_project"
```

Comment:
This shouldn't be set, since the documentation is NOT on documentation.ubuntu.com.